src/main/java/com/appium/utils/ScreenShotManager.java

```diff
@@ -89,12 +99,72 @@ public String captureScreenShot(int status, String className,
         return getDeviceModel;
     }
 
+    public String captureScreenShot(int status, String className,
+                                    String methodName, String screenShotName)
+            throws IOException, InterruptedException {
+
+        String getDeviceModel = null;
+        System.out.println("Current Running Thread Status"
+                + AppiumDriverManager.getDriver().getSessionId());
+        File scrFile = AppiumDriverManager.getDriver()
+                .getScreenshotAs(OutputType.FILE);
+        screenShotNameWithTimeStamp = currentDateAndTime();
+        if (DeviceManager.getMobilePlatform().equals(MobilePlatform.ANDROID)) {
+            getDeviceModel = screenShotNameWithTimeStamp;
+            screenShotAndFrame(status, scrFile, methodName, className, getDeviceModel,
+                    "android", screenShotName);
```

Comment:
Take off this condition and use `screenShotAndFrame(status, scrFile, methodName, className, currentDateAndTime(),DeviceManager.getMobilePlatform().toString(), screenShotName);`

Reply:
Updated